lib/event_tracer/datadog_logger.rb

```diff
@@ -64,9 +67,20 @@ def valid_args?(metrics)
     end
 
     def build_tags(args)
-      args.slice(*allowed_tags).map do |tag, value|
+      args.slice(*(allowed_tags + default_tags)).map do |tag, value|
         "#{tag}:#{value}"
       end
     end
+
+    def environment
+      ENV['APP_ENV']
+    end
+
+    def append_default_tags(args)
+      # Make sure each default tag is appended
+      @default_tags.each do |tag|
+        args[tag] = send(tag)
+      end
```

Comment:
I think this is too implicit. Would it be better to let each app input their own tags?